.github/workflows/package-linux.yml

```diff
@@ -33,9 +33,9 @@ jobs:
           { config: "Release", os: "ubuntu-22.04", arch: "arm", tls: "openssl3" },
           { config: "Release", os: "ubuntu-22.04", arch: "arm64", tls: "openssl3" },
           { config: "Release", os: "ubuntu-22.04", arch: "x64", tls: "openssl3" },
-          { config: "Release", os: "ubuntu-24.04", arch: "arm", tls: "openssl3" },
```

Comment:
Do we need to update the .azure pipelines too?

Reply:
Yes we should, I'll grab packages from GH actions first, verify the fix on docker image and will do changes on azure pipelines if it works.